nvme-print.c

```diff
@@ -5685,7 +5685,7 @@ void nvme_show_id_uuid_list(const struct nvme_id_uuid_list *uuid_list,
 				break;
 			}
 		}
-		printf(" Entry[%3d]\n", i);
+		printf(" Entry[%3d]\n", i+1);
```

Comment:
This is leads to an out of bound access. You need to update condition accordingly.

Reply:
The i+1 is only used to print the entry number.  It is not used to access any storage nor increment the value of i.  I don't see the issue here.

Reply:
Indeed. I saw something what wasn't there. I blame the weather :)